src/Pages/News/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styles from './News.module.scss';
+
+import NewsContainer from '../../components/NewsContainer';
+
+function News() {
+  const goOut = () => {
+    localStorage.removeItem('token');
+  };
+
+  return (
+    <div className={styles.news}>
+      <div className={styles.title}>News</div>
+      <button type="button" onClick={goOut}>Выйти</button>
```

Comment:
for these purposes you need to have a header component (link to main page, ligin/logout/profile buttons)